Frontend/src/components/Appointments.jsx

```diff
@@ -29,23 +38,37 @@ const Appointments = () => {
     ]);
 
     setAppointments([
-      { id: 1, patientName: "John Doe", doctorId: 1, appointmentDate: new Date(), notes: "Regular checkup", status: "Scheduled" },
-      { id: 2, patientName: "Jane Smith", doctorId: 2, appointmentDate: new Date(Date.now() + 86400000), notes: "Follow-up", status: "Scheduled" },
+      {
+        id: 1,
```

Comment:
Please use Prettier for code formatting to ensure consistency and readability.

Reply:
already used prettier for formatting and it is giving like the above only

Reply:
Chek this out!!

<img width="1031" alt="Screenshot 2024-10-23 at 4 41 59 PM" src="https://github.com/user-attachments/assets/f0d1aeaa-9203-496a-a45e-ddf645851048">